### ISSUE TITLE
Missing a Comma in CSS for e-mail button on Hover

### ISSUE THREAD

**Issue by mrnagy88** [mxstbr/sharing#56]
```
.resp-sharing-button--email:hover,
.resp-sharing-button--email:active {
  background-color: #2B2B2B !important;
  border-color: #2B2B2B !important;
}
```

The comma between the two classes is missing in the generated code so e-mail button wont change colour on hover unless you put this one in instead.

**Comment by mxstbr** [comment]
Good catch! Would you mind submitting a PR for this?

**Comment by mrnagy88** [comment]
I created a branch but not sure where the error comes from, tried searching for e-mail in all the scripts except the larger ones that need to be downloaded, I don't know if I can do this till after 5pm ;-)

**Comment by mrnagy88** [comment]
I had no idea so much can be done with JS!

**Comment by olets** [comment]
Closing: this is fixed by #59. Thanks @matter3
